<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Terzaghi1D</class>
 <widget class="QDialog" name="Terzaghi1D">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>687</width>
    <height>348</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Terzaghi 1D Solution</string>
  </property>
  <widget class="QPushButton" name="okButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>310</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Calculate and Export</string>
   </property>
  </widget>
  <widget class="QPushButton" name="closeButton">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>310</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>12</y>
     <width>669</width>
     <height>294</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Hydraulic conductivity k (m/s)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Bulk modulus K (kN/m2)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Poission's ratio</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Consolidation Coeff</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Total Consolidation Time (s)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Total Step</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Height H (m)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Z (m)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Initial Stress P0 (kPa)</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLineEdit" name="kLine">
        <property name="text">
         <string>1e-9</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="KLine">
        <property name="text">
         <string>1000</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="vLine">
        <property name="text">
         <string>0.15</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="CvLine">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="tLine">
        <property name="text">
         <string>3000</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="nsLine">
        <property name="text">
         <string>50</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="HLine">
        <property name="text">
         <string>0.0254</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="zLine">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="poLine">
        <property name="text">
         <string>100</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="../resources.qrc">:/imgs/Terzaghi.png</pixmap>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
  <zorder>label_4</zorder>
  <zorder>label_5</zorder>
  <zorder>label_6</zorder>
  <zorder>label_7</zorder>
  <zorder>label_8</zorder>
  <zorder>kLine</zorder>
  <zorder>KLine</zorder>
  <zorder>vLine</zorder>
  <zorder>CvLine</zorder>
  <zorder>tLine</zorder>
  <zorder>nsLine</zorder>
  <zorder>HLine</zorder>
  <zorder>label_9</zorder>
  <zorder>okButton</zorder>
  <zorder>closeButton</zorder>
  <zorder>zLine</zorder>
 </widget>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
